config/samples/rating_test.sh

```diff
@@ -148,6 +148,12 @@ function waitRatingDone() {
 	ratingName=$2
 	START_TIME=$(date +%s)
 	sleep 2 # wait for operator patch status. avoid 0=0 situations
+	result=$(kubectl -n${namespace} get rating ${ratingName} -ojson --ignore-not-found=true | jq -r '.status.conditions[0].status')
```

Comment:
You should check
`type`  ==   `Ready`
`Status` ==  `False`
`Message` == `Rating disabled in component's repository` 
in another function `waitRatingFail` with same `while true` loop chekcing.